CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ The changes are relative to the previous release, unless the baseline is specifi
   AVIF file has an ImageRotation property with angle set to 1 or 3, has no
   ImageMirror property, and carries an Exif chunk. Note that Exif orientation is
   usually ignored in PNG files, so this mainly impacts JPEG files.
+* Always forward Unspecified (2) CICP color primaries, transfer characteristics,
+  and matrix coefficients to the AV1 encoder. Rely on the 'colr' box instead.
```

Comment:
The commit message still has the subject line "Encode alpha with unspecified matrix coefficients". Should it be updated to match this changelog entry?

Reply:
"Encode alpha with unspecified matrix coefficients" still reflect the main intent of fixing #2685 in my opinion, but I do not mind merging this PR as "Set Unspecified CICP in Seq Header OBU".